lib/service_actor/checkable.rb

```diff
@@ -38,18 +36,14 @@ def service_actor_checks_for(origin) # rubocop:disable Metrics/MethodLength
       end
     end
 
-    def checks_class_names
-      %w[
-        TypeCheck
-        MustCheck
-        InclusionCheck
-        NilCheck
-        DefaultCheck
+    def checks_class
```

Comment:
Perhaps this should be called `check_classes`, instead :?

Reply:
It's already been done: https://github.com/sunny/actor/pull/97/commits/4743214dc4769b15fdd7687eb2260a24bd35bdf3